Cinema system/Cinema system/Models/DbModels/User.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace CinemaSystem.Models
+{
+    public class User
+    {
+        public int Id { get; set; }
+        public string Email { get; set; }
+        public string Password { get; set; }
+        public string Role { get; set; }
+        public string UserName { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }  
```

Comment:
Maybe I don’t understand what exactly, I just have validation of the fields in the RegistrationInfo and LoginInfo classes, and also automatic key generation in DBContext.

Reply:
You should have some basic validation on DB Entities like required and string length to prevent requests to DB with wrong data.
RegistrationInfo and LoginInfo classes are used in different layer (web requests). There's no guarantee that someone fill all the required data in entity before sending it to DB.